<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPEditor</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="_2">
    <item>
     <widget class="QTabWidget" name="main_tab_widget_">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="node_tab_">
       <attribute name="title">
        <string>Node</string>
       </attribute>
       <layout class="QVBoxLayout" name="node_tab_layout_">
        <item>
         <widget class="QWidget" name="node_pose_widget_" native="true">
          <layout class="QVBoxLayout" name="node_pose_layout_" stretch="0,0">
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="layout_pos_" stretch="0,0,0,0">
             <item>
              <widget class="QLabel" name="lable_pos_">
               <property name="text">
                <string>Pos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_pos_x_"/>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_pos_y_"/>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_pos_z_"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_hpr">
             <item>
              <widget class="QLabel" name="label_hpr">
               <property name="text">
                <string>HPR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_hpr_h_"/>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_hpr_p_"/>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_hpr_r_"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file_">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_project"/>
    <addaction name="separator"/>
    <addaction name="action_save_project"/>
    <addaction name="action_save_project_as"/>
    <addaction name="separator"/>
    <addaction name="action_import_model_"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help_">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about_qt"/>
    <addaction name="action_about_application"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_ffmpeg"/>
    <addaction name="action_toggle_system_console_"/>
    <addaction name="separator"/>
    <addaction name="action_options"/>
   </widget>
   <widget class="QMenu" name="menu_project">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="action_run_renderer"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_refresh_scenegraph_"/>
   </widget>
   <addaction name="menu_file_"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_project"/>
   <addaction name="menuTools"/>
   <addaction name="menu_help_"/>
  </widget>
  <widget class="QToolBar" name="tool_bar_main">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="dock_widget_project">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>145</width>
     <height>188</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tab_widget_">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="scenegraph_tab_">
        <attribute name="title">
         <string>Scene</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTreeWidget" name="scenegraph_tree_">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit this program.</string>
   </property>
  </action>
  <action name="action_open_project">
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
   <property name="toolTip">
    <string>Open project dialog.</string>
   </property>
  </action>
  <action name="action_options">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="toolTip">
    <string>Open options dialog.</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_about_application">
   <property name="text">
    <string>About RPEditor</string>
   </property>
  </action>
  <action name="action_run_renderer">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_ffmpeg">
   <property name="text">
    <string>FFmpeg</string>
   </property>
  </action>
  <action name="action_save_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="action_new_project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="action_save_project_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="action_import_model_">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Model</string>
   </property>
  </action>
  <action name="action_toggle_system_console_">
   <property name="text">
    <string>Toggle System Console</string>
   </property>
  </action>
  <action name="action_refresh_scenegraph_">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Refersh Scenegraph</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
